ui-ngx/src/app/modules/home/components/widget/lib/settings/cards/timeseries-table-widget-settings.component.html

```diff
@@ -36,6 +36,19 @@
     <mat-slide-toggle class="mat-slide"  formControlName="showMilliseconds">
       {{ 'widgets.table.display-milliseconds' | translate }}
     </mat-slide-toggle>
+    <div class="tb-form-row space-between column-xs">
+      <div translate>widgets.table.time-format</div>
+      <mat-form-field appearance="outline" subscriptSizing="dynamic">
+        <mat-select formControlName="timeFormat">
+          <mat-option [value]="'12h'">
+            {{ 'widgets.table.time-format-12h' | translate }}
+          </mat-option>
+          <mat-option [value]="'24h'">
+            {{ 'widgets.table.time-format-24h' | translate }}
+          </mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
```

Comment:
This field does not need to be displayed if we do not have a display of a column timestamp.